WalletWasabi.Fluent/Helpers/PrivacyLevelHelper.cs

```diff
@@ -0,0 +1,18 @@
+using WalletWasabi.Fluent.ViewModels.CoinSelection.Core;
+
+namespace WalletWasabi.Fluent.Helpers;
+
+internal static class PrivacyLevelHelper
+{
+	public static string GetLabelFromPrivacyLevel(PrivacyLevel privacyLevel)
+	{
+		return privacyLevel switch
+		{
+			PrivacyLevel.None => "(Invalid privacy level)",
+			PrivacyLevel.SemiPrivate => "Semi-private",
+			PrivacyLevel.Private => "Private",
+			PrivacyLevel.NonPrivate => "Unknown People",
```

Comment:
Consider getting the values from `CoinPocketHelper`... Duplication is not good.
@yahiheb What do you think saying `Private Funds` instead of `Private` still makes sense? This is the pocket name.